src/traces/scattergl/attributes.js

```diff
@@ -77,10 +81,12 @@ var attrs = module.exports = overrideAll({
     hoveron: scatterAttrs.hoveron,
 
     selected: {
-        marker: scatterAttrs.selected.marker
+        marker: scatterAttrs.selected.marker,
+        textfont: scatterAttrs.selected.textfont
     },
     unselected: {
-        marker: scatterAttrs.unselected.marker
+        marker: scatterAttrs.unselected.marker,
+        textfont: scatterAttrs.unselected.textfont
```

Comment:
Does this work? And what about `selected.textfont`?

Reply:
Confirmed. `selected.textfont.color` and `unselected.textfont.color` do **not** work.

Reply:
mostly done in https://github.com/plotly/plotly.js/commit/1a640051622f47165fd9edfa9cba073e95447f50